ci/gpu/build.sh

```diff
@@ -76,6 +76,11 @@ if [ "$(arch)" = "x86_64" ]; then
         "rapids-doc-env=${MINOR_VERSION}.*"
 fi
 
+gpuci_conda_retry remove -c conda-forge -c rapidsai -c rapidsai-nightly -c nvidia \
+      --force rapids-build-env rapids-notebook-env
+gpuci_mamba_retry install -c conda-forge -c rapidsai -c rapidsai-nightly -c nvidia \
+      treelite=3.0.0
```

Comment:
Everything looks great, just leaving a comment to delete this when the integration PR is merged

Reply:
I just merged https://github.com/rapidsai/integration/pull/532, so we can remove this and merge this PR now.